modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -192,13 +203,11 @@ $ oc apply -f <machine_config>.yml
 endif::postinstall-bare-metal[]
 
 ifdef::postinstall-bare-metal[]
-* Create a `NodeNetworkConfigurationPolicy` (NNCP) CR and define a customized `br-ex` bridge network configuration. Depending on your needs, ensure that you set a masquerade IP for either the `ipv4.address.ip`, `ipv6.address.ip`, or both parameters. Always include a masquerade IP address in the NNCP CR and this address must match an in-use IP address block.
+* Create a `NodeNetworkConfigurationPolicy` (NNCP) CR and define a customized `br-ex` bridge network configuration. The `br-ex` NNCP CR must include the OVN-Kubernetes masquerade IP address and subnet of your network. By default, the NNCP CR lists default values in the `ipv4.address.ip` and `ipv6.address.ip` parameters. You can set the masquerade IP address in the `ipv4.address.ip`, `ipv6.address.ip`, or both parameters.
```

Comment:
I don't want to bikeshed over wording, but saying "By default, the NNCP CR..." suggests to me that the CR will include the values by default, which is not true. The example NNCP CR includes the default values. There are no default behaviors on the CR itself.